src/crypto/rand/rand.h

```diff
@@ -20,7 +20,7 @@ typedef struct OQS_RAND OQS_RAND;
 /**
  * OQS PRNG object
  */
-typedef struct OQS_RAND {
+struct OQS_RAND {
```

Comment:
This should be
~~~
typedef struct OQS_RAND OQS_RAND;
struct OQS_RAND { ... };
~~~